Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6
+
+RUN echo "date.timezone = UTC" > /usr/local/etc/php/conf.d/date.ini
+
+VOLUME /app
+WORKDIR /app
+
+ENTRYPOINT ["./app/console"]
+CMD ["server:run", "0.0.0.0:8000"]
```

Comment:
Some developpers don't use the port forwarding to the Host.
But use DNS to directly access the container. It will be great to use the default port.